src/js/playlist/source.js

```diff
@@ -33,6 +33,10 @@ define([
             _source.type = extensionmap.extType(extension);
         }
 
+        if (!_source.type) {
+            return;
+        }
+
```

Comment:
Oh no! Worthy of a specific error "Could not determine media type", "{..file end string}"